rustls/src/crypto/tls13.rs

```diff
@@ -94,6 +77,21 @@ impl<'a> Hkdf for HkdfUsingHmac<'a> {
     }
 }
 
+impl<'a> HkdfPrkExtract for HkdfUsingHmac<'a> {
+    fn extract_prk_from_secret(&self, salt: Option<&[u8]>, secret: &[u8]) -> Vec<u8> {
+        let zeroes = [0u8; hmac::Tag::MAX_LEN];
+        let salt = match salt {
+            Some(salt) => salt,
+            None => &zeroes[..self.0.hash_output_len()],
+        };
+        self.0
+            .with_key(salt)
+            .sign(&[secret])
+            .as_ref()
```

Comment:
Nit: `.as_ref().into()` is a little opaque, maybe there is a more direct way to express this? Could at least use `to_vec()` or `to_owned()` instead of `into()`?

Reply:
Changed to use `to_vec()`.